1 anubis71x development

anubis71x opened the pull request:
Updated Virginia to new sail. Updated barbels to more accurate permit rudder. Fixed bug due to empty line in vessle_list. Updated all known electric torps to swimout.

IneptEuropean commented:
Are you sure all electric torps are also swim outs?

For example the Mk37 can be ejected by impulse (air) even though it's
definitely electric and a swim-out.

On 21 Aug 2017 22:56, "goldfish716" <notifications@github.com> wrote:

> Updated Virginia to new sail. Updated barbels to more accurate permit
> rudder. Fixed bug due to empty line in vessle_list. Updated all known
> electric torps to swimout.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/CaptainX3/CW-Playable-Subs/pull/38
> Commit Summary
>
>    - Virginia Model Update
>    - Merge pull request #26 from CaptainX3/0-Master-Release-Branch
>    - Barbel Rudder update.
>    - Merge branch '0-Master-Release-Branch' into 1-anubis71x-Development
>    - Electric Swimout
>    - Merge branch '0-Master-Release-Branch' into 1-anubis71x-Development
>    - Fixed end of vessels_list
>
> File Changes
>
>    - *M* vessels/_vessel_list.txt
>    <https://github.com/CaptainX3/CW-Playable-Subs/pull/38/files#diff-0>
>    (2)
>    - *M* vessels/usn_ss_barbel.txt
>    <https://github.com/CaptainX3/CW-Playable-Subs/pull/38/files#diff-1>
>    (8)
>    - *M* vessels/usn_ss_barbel_68.txt
>    <https://github.com/CaptainX3/CW-Playable-Subs/pull/38/files#diff-2>
>    (8)
>    - *M* vessels/usn_ssn_virginia.txt
>    <https://github.com/CaptainX3/CW-Playable-Subs/pull/38/files#diff-3>
>    (119)
>    - *M* weapons.txt
>    <https://github.com/CaptainX3/CW-Playable-Subs/pull/38/files#diff-4>
>    (4)
>
> Patch Links:
>
>    - https://github.com/CaptainX3/CW-Playable-Subs/pull/38.patch
>    - https://github.com/CaptainX3/CW-Playable-Subs/pull/38.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CaptainX3/CW-Playable-Subs/pull/38>, or mute the
> thread
> <https://github.com/notifications/unsubscribe-auth/Adoui9FlHZOdQ60b2lYbFq1P2VF7jstIks5safzygaJpZM4O935l>
> .
>

anubis71x commented:
Every electric I looked into has a swim out option. I assume they fire swim out at low speeds and when steady on course.

On Aug 21, 2017 6:20 PM, "CaptainX3" <notifications@github.com<mailto:notifications@github.com>> wrote:

Merged #38<https://github.com/CaptainX3/CW-Playable-Subs/pull/38>.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/CaptainX3/CW-Playable-Subs/pull/38#event-1214928244>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AZgmMyTsetd5-dZ1YcktFx1cKJ-yHstOks5sagKUgaJpZM4O935l>.

anubis71x commented:
Every electric I looked into has a swim out option. I assume that they fire swim out at low speeds and steady on course, and that they fire using conventional expulsion methods when at higher speeds and/or maneuvering.

(Resent as reply all)
anubis71x/goldfish716

On Mon, Aug 21, 2017 at 6:18 PM, IneptEuropean <notifications@github.com<mailto:notifications@github.com>> wrote:
Are you sure all electric torps are also swim outs?

For example the Mk37 can be ejected by impulse (air) even though it's
definitely electric and a swim-out.

On 21 Aug 2017 22:56, "goldfish716" <notifications@github.com<mailto:notifications@github.com>> wrote:

> Updated Virginia to new sail. Updated barbels to more accurate permit
> rudder. Fixed bug due to empty line in vessle_list. Updated all known
> electric torps to swimout.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
> https://github.com/CaptainX3/CW-Playable-Subs/pull/38
> Commit Summary
>
> - Virginia Model Update
> - Merge pull request #26 from CaptainX3/0-Master-Release-Branch
> - Barbel Rudder update.
> - Merge branch '0-Master-Release-Branch' into 1-anubis71x-Development
> - Electric Swimout
> - Merge branch '0-Master-Release-Branch' into 1-anubis71x-Development
> - Fixed end of vessels_list
>
> File Changes
>
> - *M* vessels/_vessel_list.txt
> <https://github.com/CaptainX3/CW-Playable-Subs/pull/38/files#diff-0>
> (2)
> - *M* vessels/usn_ss_barbel.txt
> <https://github.com/CaptainX3/CW-Playable-Subs/pull/38/files#diff-1>
> (8)
> - *M* vessels/usn_ss_barbel_68.txt
> <https://github.com/CaptainX3/CW-Playable-Subs/pull/38/files#diff-2>
> (8)
> - *M* vessels/usn_ssn_virginia.txt
> <https://github.com/CaptainX3/CW-Playable-Subs/pull/38/files#diff-3>
> (119)
> - *M* weapons.txt
> <https://github.com/CaptainX3/CW-Playable-Subs/pull/38/files#diff-4>
> (4)
>
> Patch Links:
>
> - https://github.com/CaptainX3/CW-Playable-Subs/pull/38.patch
> - https://github.com/CaptainX3/CW-Playable-Subs/pull/38.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CaptainX3/CW-Playable-Subs/pull/38>, or mute the
> thread
> <https://github.com/notifications/unsubscribe-auth/Adoui9FlHZOdQ60b2lYbFq1P2VF7jstIks5safzygaJpZM4O935l>
> .
>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/CaptainX3/CW-Playable-Subs/pull/38#issuecomment-323867563>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AZgmM1ZmDNJ4bFteDrK29C8eUJssk_qLks5sagJTgaJpZM4O935l>.
